src/GridFS/Bucket.php

```diff
@@ -120,6 +128,7 @@ public function __construct(Manager $manager, $databaseName, array $options = []
         $this->databaseName = (string) $databaseName;
         $this->bucketName = $options['bucketName'];
         $this->chunkSizeBytes = $options['chunkSizeBytes'];
+        $this->disableMD5 = isset($options['disableMD5']) ? $options['disableMD5'] : false;
```

Comment:
For consistency, you can add `'disableMD5' => false,` when `$options` is appended at the start of the constructor. This can then become `$this->disableMD5 = $options['disableMD5'];`. Prior art: https://github.com/mongodb/mongo-php-library/blob/baf85f46d578acf25ef11d65eb28ff5bd4b75503/src/Operation/FindAndModify.php#L114-L118